src/Tabs.re

```diff
@@ -86,7 +86,14 @@ type action =
 
 module AddTab = {
   [@react.component]
-  let make = (~dispatch: action => unit, ~depth: int, ~standalone, ~theme) => {
+  let make =
+      (
+        ~dispatch: action => unit,
+        ~depth: int,
+        ~standalone,
+        ~theme,
+        ~iconOnly=false,
```

Comment:
I would model this as something along the lines of `Option<addText>` rather than adding a boolean flag. I'm not sure if this is possible, though.

Reply:
Fair enough - that makes sense :)